src/components/CustomProperties/styles.ts

```diff
@@ -78,11 +73,4 @@ export function getCustomProperties(tokens: TokenGroup) {
 /**
  * Adapted from: https://github.com/argyleink/gui-challenges/blob/main/color-schemes/style.css
  */
-export const styles = /* css */ `
-:root {
-  ${lightDeclarations}
-  ${getStaticCustomProperties(tokens)}
-}
-
-${getColorSchemeRules(tokens, osColorSchemes)}
-`;
+export const styles = getColorSchemeRules(tokens, osColorSchemes);
```

Comment:
I wonder if we should still apply the default color-scheme to `:root`. This was adapted from the [gui-challenges/color-schemes](https://github.com/argyleink/gui-challenges/blob/main/color-schemes/style.css) and I think we may need to have a set of custom properties defined at the highest layer. I haven't thought through all the edge cases, and I'm sure there out there, but I could see this being an issue where a component is rendered in a portal (outside the React DOM hierarchy) and doesn't have access to any CSS vars.